torchvision/models/detection/fcos.py

```diff
@@ -502,7 +502,8 @@ def postprocess_detections(
 
                 # keep only topk scoring predictions
                 num_topk = det_utils._topk_min(topk_idxs, self.topk_candidates, 0)
-                scores_per_level, idxs = scores_per_level.topk(num_topk)
+                scores_per_level, idxs = scores_per_level.sort(dim=-1, descending=True, stable=True)
+                scores_per_level, idxs = scores_per_level[:num_topk], idxs[:num_topk]
```

Comment:
topk sorts by default so this should not be slower than what we currently do but needs to be benchmarked